external/thorntail-mp-tck/playlist.xml

```diff
@@ -12,10 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 -->
+<!-- Tests require a larger java heap than the default when running with openj9 jdk.  -->
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TestConfig/playlist.xsd">
 	<test>
 		<testCaseName>thorntail_microprofile_tck</testCaseName>
-		<command>docker run --name thorntail-mp-tck $(EXTRA_DOCKER_ARGS) adoptopenjdk-thorntail-mp-tck:latest \
+		<command>docker run --name thorntail-mp-tck --env OPENJ9_JAVA_OPTIONS=-Xmx1g $(EXTRA_DOCKER_ARGS) adoptopenjdk-thorntail-mp-tck:latest \
```

Comment:
This will also inject the environment variable into the docker environment running against Hotspot, which won't hurt anything. However  we could also pass through --build-arg values based on JDK_IMPL when running the docker build. And then ENV the passed in build-arg values.

Reply:
Yes I did realise.  As you say, it will be ignored by hotspot.  Done this way it keeps the code change to a minimum.